frontend/public/components/ingress.jsx

```diff
@@ -199,47 +169,87 @@ const RulesRows = (props) => {
   return <EmptyBox label="Rules" />;
 };
 
-const Details = ({ obj: ingress }) => (
-  <>
-    <div className="co-m-pane__body">
-      <SectionHeading text="Ingress Details" />
-      <ResourceSummary resource={ingress}>
-        <dt>TLS Certificate</dt>
-        <dd>{getTLSCert(ingress)}</dd>
-      </ResourceSummary>
-    </div>
-    <div className="co-m-pane__body">
-      <SectionHeading text="Ingress Rules" />
-      <p className="co-m-pane__explanation">
-        These rules are handled by a routing layer (Ingress Controller) which is updated as the
-        rules are modified. The Ingress controller implementation defines how headers and other
-        metadata are forwarded or manipulated.
-      </p>
-      <div className="co-m-table-grid co-m-table-grid--bordered">
-        <RulesHeader />
-        <RulesRows spec={ingress.spec} namespace={ingress.metadata.namespace} />
+const Details = ({ obj: ingress }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('ingress~Ingress details')} />
+        <ResourceSummary resource={ingress}>
+          <dt>{t('ingress~TLS certificate')}</dt>
+          <dd>
+            <TLSCert ingress={ingress} />
+          </dd>
+        </ResourceSummary>
       </div>
-    </div>
-  </>
-);
-
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('ingress~Ingress rules')} />
+        <p className="co-m-pane__explanation">
+          {t(
+            'ingress~rules_description',
+            'These rules are handled by a routing layer (Ingress Controller) which is updated as the rules are modified. The Ingress controller implementation defines how headers and other metadata are forwarded or manipulated',
```

Comment:
I'm concerned using keys and default values will make it difficult to pass files off to translation. (They pretty much only get the keys.) Do you have thoughts on that @spadgett? Thanks.

Reply:
I thought this was neccesary due to some character limit of the keys?

Reply:
I believe that was actually an issue with i18n not being configured to allow periods. We had the parser configured, but not i18n itself. I think Robb said it was resolved.

Reply:
Let me know if you run into something weird like that though.

Reply:
Originally we had a problem with periods, but I thought that was resolved. @rhamilto, did you switch to using the full string as a key?

The translators should have access to the English file, so I think they could figure it out. But I might just use the full message as the key everywhere so we're consistent and there's no confusion.

Reply:
Changed to use full message as the key.